app/scripts/templates/user_pages/item.html

```diff
@@ -1,4 +1,4 @@
-<a class="image {{imagePosition}}" href="{{url}}">
+<a class="image {{imagePosition}}" href="{{_getUrl}}">
```

Comment:
This looks weird, why the underscore in `{{_getUrl}}`? The underscore makes me think its a private variable or something.